papas/graphtools/graphbuilder.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, ids, edges):
         '''
         self.ids = ids
         self.edges = edges
-
```

Comment:
I think this is a good idea.
I have renamed GraphBuilder to SubgraphBuilder for now.